collectors/pool_usage.go

```diff
@@ -55,6 +59,15 @@ func NewPoolUsageCollector(conn Conn) *PoolUsageCollector {
 			},
 			[]string{"pool"},
 		),
+		MaxAvail: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: cephNamespace,
+				Subsystem: "df",
```

Comment:
As long the namespace is clear in terms of what we are tracking and removes redundant prefix from appearing into metric names I think we should be good. My aversion to `df` is that it is slightly more specific to be a part of a subsystem namespace since (a) there are multiple `df`s in ceph (there's `ceph df` and `rados df`) and (b) it only highlights the usage in terms of bytes. `pool` probably seems more ideal since we already use it in our names.

We can include additional statistics related to pool-specific state changes but if they are not related to usage then they should be in a separate unit ideally. A PR is most welcome! :)